tests/ui/impl-trait/where-allowed.rs

```diff
@@ -155,97 +155,97 @@ type InTypeAlias<R> = impl Debug;
 //~^ ERROR `impl Trait` in type aliases is unstable
 
 type InReturnInTypeAlias<R> = fn() -> impl Debug;
-//~^ ERROR `impl Trait` only allowed in function and inherent method argument and return types
+//~^ ERROR `impl Trait` is not allowed in `fn` pointer
```

Comment:
`` `fn` pointers `` plural sounds better I guess. Odd that we don't say `` `fn` pointer return types `` here while we do say that in other places

Reply:
sorry lol we do say the whole thing, i just only matched a subset of the error message